src/main/java/gregtech/api/util/ItemStackHashStrategy.java

```diff
@@ -58,7 +58,7 @@ static ItemStackHashStrategy comparingItemDamageCount() {
      */
     class ItemStackHashStrategyBuilder {
 
-        private boolean item, count, damage, tag;
+        private boolean item, count, damage, tag, meta;
```

Comment:
Why is damage not enough?

Reply:
iirc because of GT Tools or something like that, i made this change a long time ago.